tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -563,6 +563,49 @@ void test_gij_deriv_functions_analytic(
   CHECK_ITERABLE_APPROX(d_spatial_metric_expected, d_gij);
   CHECK_ITERABLE_APPROX(d4_g_expected, d4_g);
 }
+
+template <typename DataType, size_t SpatialDim, typename Frame>
+void test_spacetime_derivative_of_spacetime_metric(
+    const DataVector& used_for_size) noexcept {
+  CAPTURE(SpatialDim);
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> distribution(0.1, 1.0);
+
+  const auto lapse = make_with_random_values<Scalar<DataType>>(
+      make_not_null(&generator), make_not_null(&distribution), used_for_size);
```

Comment:
can you just use a `size_t` rather than the `used_for_size`? I forget which of those I have and have not made.